src/Theme/utils.ts

```diff
@@ -123,3 +123,9 @@ export enum TEXT_VARIANT {
   NEGATIVE = 'negative',
   BASE = 'base',
 }
+
+export enum ARIA_LIVE_VARIANT {
```

Comment:
I don't think `Theme/utils` is the right place for this, since it's not really related to the theme. What would you think about adding a `const.ts` file at the top level for this, and any future non-theme constants?